types/v1/metrics.go

```diff
@@ -0,0 +1,24 @@
+package v1
+
+import "github.com/rancher/go-rancher/client"
+
+// OpenEBSVolumeMetrics is used to store the collected metrics
+// all the stats exposed by jiva stored into OpenEBSVolumeMetrics fields
+type OpenEBSVolumeMetrics struct {
+	Resource        client.Resource
```

Comment:
Are we really really dependent on rancher... It will be great to remove any dependencies from types package.....

Reply:
@AmitKumarDas you said that reuse the struct already declared so that it can be used by other code also. So i copied this struct from VolumeStats in volume_stats.go so that it can be reused